### PR TITLE
docs: update DatePicker, Drawer and FileInput component

### PR DESCRIPTION
**Pull request by Annaleah123**
### PR Checklist

Please check if your PR fulfills the following requirements:

-   [ ] Tests for the changes have been added (for bug fixes/features)
-   [x] Docs have been added/updated (for bug fixes/features)

### PR Type

What kind of change does this PR introduce?

-   [ ] Bugfix
-   [ ] Feature
-   [ ] Code style update (formatting, local variables)
-   [ ] Refactoring (no functional changes, no api changes)
-   [ ] Build related changes
-   [ ] CI related changes
-   [ ] Performance improves
-   [ ] Adding missing tests
-   [x] Documentation content changes
-   [ ] Other... Please describe:

### Does this PR introduce a breaking change?

-   [ ] Yes
-   [x] No

### Additional context

Provide any additional context or screenshots about the feature PR.

### CODE REVIEW

File: packages/core/src/components/FileInput/types.ts [hunk 1]
```diff
@@ -24,7 +24,7 @@ export interface FileInputProps extends Omit<HTMLProps<HTMLInputElement>, 'type'
     fullWidth?: boolean;
     /** Disable Input */
     disabled?: boolean;
-    /** To be used when it is required in any form */
+    /** To use as per requirement in any form */
```
Comment (wsameer) [inline packages/core/src/components/FileInput/types.ts:27]: @Annaleah123  we could rephrase this as, "Use to make the input field mandatory while filing the form"

File: packages/core/src/components/DatePicker/DatePicker.stories.mdx [hunk 2]
```diff
@@ -18,18 +18,21 @@ import { placements } from '../Popover/Popover.stories.tsx';
     }}
 />
 
-# DatePicker component
+# DatePicker Component
 
-The datepicker component allows you to select a date through input text or calendar date selection. 
+The DatePicker allows you to select a date through input text or using a calendar dialog.
```
Comment (abhishekashyap) [inline packages/core/src/components/DatePicker/DatePicker.stories.mdx:23]:
```suggestion
The `DatePicker` allows you to select a date through input text or using a calendar dialog. 
```

**Review by abhishekashyap — CHANGES_REQUESTED** [review]

File: packages/core/src/components/DialogBox/DialogBox.stories.mdx [hunk 3]
```diff
@@ -6,9 +6,9 @@ import * as stories from './DialogBox.stories';
 
 # DialogBox
 
-Basic DialogBox component. You can play with the component in the canvas tab.
+Dialog boxes are windows that appear in front of app content to give critical information, request a decision, or involve multiple tasks.  It consists of a Header, Content, and Action.
```
Comment (abhishekashyap) [inline packages/core/src/components/DialogBox/DialogBox.stories.mdx:9]:
```suggestion
`DialogBox` are windows that appear in front of app content to give critical information, request a decision, or involve multiple tasks.  It consists of a Header, Content, and Action.
```

File: packages/core/src/components/DatePicker/DatePicker.stories.mdx [hunk 4]
```diff
@@ -18,18 +18,21 @@ import { placements } from '../Popover/Popover.stories.tsx';
...
 />
 
-# DatePicker component
+# DatePicker Component
 
-The datepicker component allows you to select a date through input text or calendar date selection. 
+The `DatePicker` allows you to select a date through input text or using a calendar dialog. 
 
-It is a controlled react component and must have the following handle state:
+Handle the state as follows:
```
Comment (gmukul01) [inline packages/core/src/components/DatePicker/DatePicker.stories.mdx:25]: We should inform the user that `Its a controlled component`

File: packages/core/src/components/DatePicker/DatePicker.stories.mdx [hunk 5]
```diff
@@ -18,18 +18,21 @@ import { placements } from '../Popover/Popover.stories.tsx';
...
 ```tsx
 const [date, setDate] = useState<Date | null>(null);
 return <DatePicker label="Select Date" value={date} onChange={setDate} />;
 ```
 
-To open the calendar, click on the calendar icon inside the DatePicker component. To close the calendar, select any date or click outside of the DatePicker component. You can also manually enter the date.
+To use the calendar below, follow the steps given:
+1. Select the calendar icon inside the DatePicker component to open the calendar.
+2. Select any date or click outside the DatePicker component to close the calendar.
+3. The provision to enter the date manually is applicable.
```
Comment (gmukul01) [inline packages/core/src/components/DatePicker/DatePicker.stories.mdx:35]: I don't think `applicable` is a correct word over here

File: packages/core/src/components/DialogBox/DialogBox.stories.mdx [hunk 6]
```diff
@@ -22,25 +22,32 @@ DialogBox consists of three sections Header, Content and Actions. All of these y
...
+Shadow appears on the Header and Actions section if you scroll through the content.
 
-1. Click on the cross icon
-2. Press `Esc` key
-3. If you have provided `shouldCloseOnOutsideClick` this prop then dialog-box will close on clicking outside of dialog-box
+To close the dialog box, follow the steps given below:
+
+1. Click the cross icon.
+2. Press `Esc` key.
+3. If the `shouldCloseOnOutsideClick` prop is selected, then dialog box will close on clicking outside the dialog box.
```
Comment (gmukul01) [inline packages/core/src/components/DialogBox/DialogBox.stories.mdx:31]: I think `selected` is not the right word over here, we can use `provided` or `set true`

File: packages/core/src/components/Drawer/Drawer.stories.mdx [hunk 7]
```diff
@@ -6,7 +6,7 @@ import * as stories from './Drawer.stories';
 
 # Drawer
 
-Drawer consists of three sections Header, Content and Footer. All of these you can directly use from Drawer component only like below ...
+A `Drawer` component provides access to locations in your application. It is anchored to the left or right edge of the screen. It consists of a Header, Content, and Footer.
```
Comment (gmukul01) [inline packages/core/src/components/Drawer/Drawer.stories.mdx:9]: `A Drawer component provides access to locations in your application.` This is not correct.

File: packages/core/src/components/Drawer/Drawer.stories.mdx [hunk 8]
```diff
@@ -20,20 +20,26 @@ Drawer consists of three sections Header, Content and Footer. All of these you c
...
+## Example
+
+Use the example given below: 
 
 <Preview withToolbar>
     <Story name="Drawer">{stories.Basic()}</Story>
 </Preview>
 
-> Note: Check the detailed code at this [link](https://github.com/medly/medly-components/blob/master/packages/core/src/components/Drawer/Drawer.stories.tsx)
+> Note: To view a complete description of the code click [here](https://github.com/medly/medly-components/blob/master/packages/core/src/components/Drawer/Drawer.stories.tsx)
```
Comment (gmukul01) [inline packages/core/src/components/Drawer/Drawer.stories.mdx:33]: its not the description, its the code itself

File: packages/core/src/components/FileInput/FileInput.stories.mdx [hunk 9]
```diff
@@ -7,16 +7,19 @@ import * as stories from './FileInput.stories';
 
 # FileInput
 
-### Basic FileInput component
+The `FileInput` component enables uploading files from the local storage to the server via the File API. As its value is read-only, it is an uncontrolled component.
```
Comment (gmukul01) [inline packages/core/src/components/FileInput/FileInput.stories.mdx:10]: We can just keep `The FileInput component enables uploading files from the local storage.`

**Review by gmukul01 — CHANGES_REQUESTED** [review]

File: packages/core/src/components/DatePicker/DatePicker.stories.mdx [hunk 10]
```diff
@@ -18,18 +18,21 @@ import { placements } from '../Popover/Popover.stories.tsx';
...
+Handle the state as follows:
 
 ```tsx
 const [date, setDate] = useState<Date | null>(null);
 return <DatePicker label="Select Date" value={date} onChange={setDate} />;
 ```
 
-To open the calendar, click on the calendar icon inside the DatePicker component. To close the calendar, select any date or click outside of the DatePicker component. You can also manually enter the date.
+To use the calendar below, follow the steps given:
+1. Select the calendar icon inside the DatePicker component to open the calendar.
```
Comment (gmukul01) [inline packages/core/src/components/DatePicker/DatePicker.stories.mdx:33]: These three are not steps, they are individual statements. Also instead of using `select` word we should use `click`.

**Review by gmukul01 — CHANGES_REQUESTED** [review]
